### ISSUE TITLE
Valeur par défaut climatisation

### ISSUE THREAD

**Issue by florianpanchout** [incubateur-ademe/nosgestesclimat#2052]
[25 % des Français ont un climatiseur](https://www.actu-environnement.com/media/pdf/news-37991-ademe-etude-climatisation-france-2020.pdf). Je questionne donc la pertinence d'afficher oui par défaut.

Et je questionne encore plus le 0,3 climatiseur par défaut...

**Comment by JuliePouliquen** [comment]
Très bonne remarque qui renvoie à #1907 il me semble.

**Comment by Clemog** [comment]
Si on ne met pas "oui" par défaut, la réponse au nombre de climatiseurs ne sera plus 0,3 mais 0..
Lié à https://github.com/incubateur-ademe/nosgestesclimat/issues/1907

**Comment by florianpanchout** [comment]
Si on mets non par défaut et que l'utilisateur répond oui, il aura quand même 0,3 qui s'affiche non ?

**Comment by Clemog** [comment]
Yes mais dans le calcul de l'empreinte "moyenne", la climatisation sera à 0 alors que c'est 0,3

**Comment by florianpanchout** [comment]
On est d'accord qu'il y a un problème de consistance sur les valeurs par défaut non ? Certaines questions (comme celle-ci) font une moyenne et d'autres prennent l'option la plus répandue (comme la question sur les bouteilles d'eau).

**Comment by Clemog** [comment]
Here it is : https://pad.incubateur.net/0LHEec49RYWJXLBqHCpC2w
